dashboard/app/assets/stylesheets/ai_tutor_panel.scss

```diff
@@ -0,0 +1,71 @@
+@import "color";
+@import "font";
```

Comment:
- [ ] lightweight styling to include A.I. Bot image and differentiate from teacher panel